### ISSUE TITLE
Add some input for a comment to add to the log

### ISSUE THREAD

**Issue by netizen-ais** [meyercm/atom-work-log#1]
It would be very handy to set an 'edit reason' that lasts for current session (list a commit message).

**Comment by meyercm** [comment]
Thanks for the input- I hope that the package is of use so far!

How do you see your 'edit reason' working? Specifically, what do you think about the following:

* How much room for input (1 line vs. many)?
* How often to prompt (i.e. what is a 'session' from your description?)?
* How to store that info?

If I were building that feature without any input from you, I think I'd pop up an input after some period of inactivity; in pseudocode:

```pseudo
on(:save, file) {
  if (now() - lastsave) > 20.minutes  { 
    reason = prompt("what are you working on?")
    if reason.trim() != "" { 
      log.write("#{timestamp} #{reason}") 
    }
  }
  log.write("#{timestamp} #{file}")
  lastsave = now()
}
```
In plain english: if you haven't saved for 20 minutes or more, then when you do save, show the pop-up and solicit input e.g. 'what are you working on?'. In terms of storage, I'd probably just add the resulting string to the log with a timestamp, but I'm open to other ideas if you have them.

Thoughts?

**Comment by netizen-ais** [comment]
Optimally I would set a command to invoke the input. I would then use that until I change it invoking the command again.

**Comment by netizen-ais** [comment]
I forgot to answer your other questions:
I was thinking about it being 1 single line, and to store it next to the filename in the log:

timestamp /path/file what_i_was_doing
